source/develop/release-management/features/network/firewalld-support.html.md

```diff
@@ -0,0 +1,65 @@
+---
+title: firewalld-support
+category: feature
+authors: Leon Goldberg
+wiki_category: Feature
+wiki_title: Features/firewalld-support
+wiki_revision_count: 1
+wiki_last_updated: 2017-04-18
+feature_name: Firewalld Support
+feature_modules: Networking
+feature_status: POC
+---
+
+# Firewalld Support
+
+### Owner
+
+*   Name: Leon Goldberg (lgoldber)
+*   Email: lgoldber@redhat.com
+
+## Summary
+
+firewalld is a modern and natively distributed (CentOS 7+, Fedora 18+, RHEL 7+) firewall management solution. Essentially firewalld is an iptables abstraction,
```

Comment:
Not to block this PR, just for curiosity: is firewalld just an ip(6)tables abstraction? I would expected it utilize ebtables and nftables, too.